certify-service/src/main/resources/application-local.properties

```diff
@@ -171,6 +171,7 @@ mosip.certify.key-values={\
               }}\
    }\
 }
```

Comment:
Please sync back the changes to this file to relevant configs in `docker-compose/` directory once the naming is resolved and agreed upon.

Reply:
Added changes to certify-default properties